mllib/src/main/scala/org/apache/spark/ml/param/sharedParams.scala

```diff
@@ -23,6 +23,27 @@ private[ml] trait HasRegParam extends Params {
   def getRegParam: Double = get(regParam)
 }
 
+private[ml] trait HasElasticNetParam extends HasRegParam {
+  /** param for elastic net regularization parameter */
+  val alphaParam: DoubleParam = new DoubleParam(
+    this, "alphaParam", "the ElasticNet mixing parameter")
+  def getAlphaParam: Double = get(alphaParam)
+}
+
+private[ml] trait HasConvergenceTol extends Params {
```

Comment:
In `spark.ml`, we try to use short names for parameters. This could be called `tol`, as used in sklearn and matlab.